examples/kotlinExample/build.gradle

```diff
@@ -1,7 +1,8 @@
 buildscript {
-    ext.kotlin_version = '1.0.0-rc-1036'
+    ext.kotlin_version = '1.0.0'
     repositories {
         jcenter()
+        mavenCentral()
```

Comment:
It seems a bit superfluous, but I don't have any strong opinions on it. So if you think it makes it easier for people to compare their project with our example, just keep it.